colors/bearded-theme-vivid-black.lua

```diff
@@ -0,0 +1,69 @@
+-- Theme by @xStormyy on GitHub
+-- Original Color Scheme: https://github.com/BeardedBear/bearded-theme/blob/master/themes/bearded-theme-vivid-black.json
+
+
+local style = require "core.style"
+local common = require "core.common"
+
+
+-- base colors
+--[[
+red = "#D62C2C"
+orange = "#FF7135"
+yellow = "#FFB638"
+green = "#42DD76"
+blue = "#28A9FF"
+cyan = "#14E5D4"
+magenta = "#E66DFF"
+white = "#c8c8c8"
+black = "#141417"
+]]--
+
+--[[
+    #////////////////////////////#
+           User Interface
+    #////////////////////////////#
+]]--
+style.background2 = { common.color "#0f0f11" } -- sidebar
+style.background3 = { common.color "#0f0f11" } -- status bar
+style.text = { common.color "#afafafCC" } -- user interface text
+
+style.accent = style.caret -- accent color
+style.divider = { common.color "#060607" } -- borders
+
+style.line_number = { common.color "#55555590" } -- inactive
+style.line_number2 = { common.color "#555555" } -- active
+
+
+--[[
+    #////////////////////////////#
+                Editor
+    #////////////////////////////#
+]]--
+style.background = { common.color "#141417" } -- editor
+style.syntax["normal"] = { common.color "#c8c8c8" } -- editor text
+
+style.syntax["symbol"] = { common.color "#c8c8c8" }  -- symbols
+style.syntax["comment"] = { common.color "#555555" } -- comments
+style.syntax["keyword"] = { common.color "#28A9FF" }  -- keywords like function, local, if, end, in lua
+style.syntax["keyword2"] = { common.color "#E66DFF" } -- types
+style.syntax["keyword3"] = { common.color "#FF000FF" } -- absolutely no idea
```

Comment:
I don't think this is a thing, did you find anything that uses it?

Reply:
I found those in some community color schemes

Reply:
Doesn't hurt to have it, it just won't be used as I'm not aware of any language plugin that uses it.